test/UnitTest/Microsoft.AspNet.OData.Test.Shared/Formatter/ODataActionTests.cs

```diff
@@ -74,10 +74,10 @@ public async Task CanDispatch_ActionPayload_ToBoundAction()
         }
 
         private const string EntityPayload = @"{ 
-                ""Customer"": {""@odata.type"":""#Microsoft.AspNet.OData.Test.Formatter.Customer"", ""ID"":101,""Name"":""Avatar"" } , 
+                ""Customer"": {""@odata.type"":""#org.odata.Customer"", ""ID"":101,""Name"":""Avatar"" } , 
```

Comment:
>org.odata [](start = 50, length = 9)

Would prefer to use a namespace besides org.odata, which sounds official. maybe something like My.Schema?